src/Hooks/fetchAllDataHook.js

```diff
@@ -0,0 +1,12 @@
+// import { useCallback } from 'react';
+// import * as API from '../api/apis';
+
+// export const useFetchAllDataHooks = () => {
+//   const getJob = useCallback(() => {
+//     API.newJobs().then((res) => {
+//       console.log(res);
+//     });
+//   }, []);
+
+//   return { getJob };
+// };
```

Comment:
- [ ] Kindly, remove the commented code present in this file in order to improve the repo's cleanliness. 👍